vite.config.js

```diff
@@ -14,6 +15,7 @@ export default defineConfig({
     eslintPlugin({
       cache: false, // disable eslint cache to avoid conflicts
     }),
+    nodePolyfills(),
```

Comment:
Are we able to describe what is the issue? We could reference not only issue but also what causes it

It seems that it is again related to lib `parse-link-headers`. We can discuss alternatives to this library. Maybe some fixed fork?

Reply:
again we need to reference here issue which we can watch to solve the issue